### ISSUE TITLE
Keyboard Navigation

### ISSUE THREAD

**Issue by apdonaghy** [lukeware/happy-pixies#3]
## User story: [Keyboard navigation](https://github.com/lukeware/happy-pixies/wiki/User-Story:-Keyboard-navigation)
**Size** (md)

**What is the priority that has been assigned?**
High

**How do you interpret this fitting into the process of building the project?**
I see this fitting in at the beginning stages when there is even the smallest amount of content to work with, but then added to as the build becomes more robust. 

**Why should this be in the Sprint this time? Why not wait until the next Sprint?**
Accessibility features should be implemented at the beginning of all projects and then continuously updated since it's more difficult to add these features when the site/app is finished.

**Comment by apdonaghy** [comment]
**Task branch name**
keyboard-navigation

**Brief statement of work performed**
Added todos for future sprints

**Any notes, comments, or update instructions about the work.**
Hoping to start working on accessibility features in the next sprint
